cypress.json

```diff
@@ -1,8 +1,9 @@
 {
-  "baseUrl": "http://localhost:3000",
+  "baseUrl": "http://localhost:9081",
```

Comment:
not meshery.layer5.io, right?

Reply:
No it's the URL on which the cypress -ui tests will be done which is the meshery-localhost.